Automattic/jetpack
# Social Logos: New logos and various fixes

This PR adds several new logos that were requested in the old repo during cleanup. While adding them, I realised there were a few bugs introduced in #37528 that had to be fixed at the same time.

## Proposed changes:

This adds the following icons:
* Deezer: #37550
* Discord: #37549
* git: #37548
* LINE: #37552
* Messenger: #37551
* Quora: #37545
* Snapchat: #37539
* Soundcloud: #37538
* Untappd: #37546
* VK: #37543

When adding these, I repaired few regressions:

* The fonts build wasn't working because `glob 9` had changed its API (we were using glob 7, and it was updated to glob 10).
* File sorts were relying on locale; for consistency we now rely on the default JS sort.
* The example React component wasn't copying the CSS into place. I now centralized the source CSS file (as it was in an earlier iteration) and copy it to both `./build/react/` and `./build/svg-sprite/`.

Finally, there's slightly better error-handling; the script returns a non-zero code if the build chain fails.

And post-finally:
* The license section of the `README.md` was updated.
* Each icon family has an entry in the new `official_links.md` file.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Make sure CI is happy.
* Run `pnpm build`, then open `./build/svg-sprite/example.html` in a browser to verify the icons show correctly.
